service/test/ConstConfigIOGroupTest.cpp

```diff
@@ -215,14 +251,18 @@ TEST_F(ConstConfigIOGroupTest, input_empty_domain)
     create_config_file(json_string);
 
     GEOPM_EXPECT_THROW_MESSAGE(
-        ConstConfigIOGroup iogroup(M_CONFIG_FILE_PATH, ""),
+        ConstConfigIOGroup iogroup(*m_default_topo, M_CONFIG_FILE_PATH, ""),
         GEOPM_ERROR_INVALID,
         "PlatformTopo::domain_name_to_type(): unrecognized domain_name: "
     );
 }
 
 TEST_F(ConstConfigIOGroupTest, input_empty_description)
 {
+    EXPECT_CALL(*m_default_topo, num_domain(_)).Times(0);
+    EXPECT_CALL(*m_default_topo, num_domain(GEOPM_DOMAIN_GPU))
+        .Times(AtMost(1))
+        .RetiresOnSaturation();
```

Comment:
This appears to be used by a large number of tests, may be something we can refactor, but it isn't necessarily a high priority.